osu.Game.Rulesets.Osu/Objects/SliderTailCircle.cs

```diff
@@ -9,16 +9,28 @@ namespace osu.Game.Rulesets.Osu.Objects
 {
     public class SliderTailCircle : SliderEndCircle
     {
+        /// <summary>
+        /// Whether to treat this <see cref="SliderHeadCircle"/> as a normal <see cref="HitCircle"/> for judgement purposes.
+        /// If <c>false</c>, this <see cref="SliderHeadCircle"/> will be judged as a <see cref="SliderTick"/> instead.
+        /// </summary>
+        public bool ClassicSliderBehaviour;
```

Comment:
Flag should work like the `SliderHeadCircle` flag was made to in bf9f20705f06e3094ae7e772f4ac59fa45472856, i.e. it should only be required to set it on the slider.

In fact:

![image](https://github.com/ppy/osu/assets/20418176/de444893-fcde-4631-a064-28e3e2b6bb2a)